org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/util/Icon.java

```diff
@@ -0,0 +1,31 @@
+/******************************************************************************
+ * Copyright (C) 2024 Zuhri Utama. A                                     *
+ * This program is free software; you can redistribute it and/or modify it    *
+ * under the terms version 2 of the GNU General Public License as published   *
+ * by the Free Software Foundation. This program is distributed in the hope   *
+ * that it will be useful, but WITHOUT ANY WARRANTY; without even the implied *
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.           *
+ * See the GNU General Public License for more details.                       *
+ * You should have received a copy of the GNU General Public License along    *
+ * with this program; if not, write to the Free Software Foundation, Inc.,    *
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.                     *
+ *****************************************************************************/
+package org.adempiere.webui.util;
+
+import java.io.Serializable;
+
+public class Icon implements Serializable {
```

Comment:
It is a class with only constant, should be final (i.e `public final class`) and shouldn't implement Serializable.

Also, should have a private constructor (`private Icon() {}`) since you would never wants to create a new instance of this class.